packages/core/src/shared/errors.ts

```diff
@@ -15,6 +15,7 @@ import type * as os from 'os'
 import { CodeWhispererStreamingServiceException } from '@amzn/codewhisperer-streaming'
 import { driveLetterRegex } from './utilities/pathUtils'
 import { getLogger } from './logger/logger'
+import * as CrashMonitoring from './crashMonitoring/constants'
```

Comment:
the main constants.ts file generally exists for this reason. we don't really want errors.ts importing a bunch of random modules (partly because of the circular dep you ran into, but also because it's a low-level module that should be importable ~everywhere)

 do these constants need their own crashmonitoring/constants.ts  module or can they just live in the existing constants.ts module

Reply:
good point with the importable everywhere. I've moved these to the shared constants file